app/code/community/Bolt/Boltpay/Model/FeatureSwitch.php

```diff
@@ -29,6 +29,50 @@ class Bolt_Boltpay_Model_FeatureSwitch extends Bolt_Boltpay_Model_Abstract
      */
     public static $shouldUpdateFeatureSwitches = false;
 
+    // Switches field names
+    const VAL_KEY = 'value';
+    const DEFAULT_VAL_KEY = 'defaultValue';
+    const ROLLOUT_KEY = 'rolloutPercentage';
+
+    /**
+     * This switch is a sample of how to set up a feature switch.
+     * Every feature switch added here should have a corresponding helper
+     * in this class
+     */
+    const SAMPLE_SWITCH_NAME = 'M1_SAMPLE_SWITCH';
+    const BOLT_ENABLED_SWITCH_NAME = 'M1_BOLT_ENABLED';
+
+    const COOKIE_NAME = 'BoltFeatureSwitchId';
+
+    /**
+     * @var array Bolt features.
+     */
+    private $switches;
+
+    /**
+     * @var array Default values for Bolt features
+     */
+    private $defaultSwitches;
+
+    /**
+     * Bolt_Boltpay_Model_FeatureSwitch constructor.
+     */
+    final public function __construct()
```

Comment:
Let's write a test that asserts $this->defaultSwitches gets set to the correct value.